src/v/coproc/tests/router_test.cc

```diff
@@ -62,9 +62,8 @@ FIXTURE_TEST(test_coproc_router_no_results, router_test_fixture) {
 FIXTURE_TEST(test_coproc_router_simple, router_test_fixture) {
     auto client = make_client();
     client.connect().get();
-    // Supervisor has 3 registered transforms, of the same type
+    // Supervisor has 2 registered transforms
     add_copro<identity_coprocessor>(1234, {{"foo", l}}).get();
-    add_copro<identity_coprocessor>(121, {{"foo", l}}).get();
```

Comment:
after some discussion this isn't a bug and should work, maybe rename the commit and fix later: #380